app/services/atmosphere/nic_provider/null_nic_provider.rb

```diff
@@ -0,0 +1,9 @@
+module Atmosphere
+  module NicProvider
+  class NullNicProvider
+    def get(appl, tmpl)
```

Comment:
Unused method argument - `appl`. If it's necessary, use `_` or `_appl` as an argument name to indicate that it won't be used. You can also write as `get(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `tmpl`. If it's necessary, use `_` or `_tmpl` as an argument name to indicate that it won't be used. You can also write as `get(*)` if you want the method to accept any arguments but don't care about them.